core/filepicker.go

```diff
@@ -731,7 +733,7 @@ func (fb *FileButton) Init() {
 		if fb.Filename == "" {
 			fb.SetText("Select file")
 		} else {
-			fb.SetText(fb.Filename)
+			fb.SetText(elide.Middle(fb.Filename, 38))
```

Comment:
so just a magic number 38 without any global var to override?

Reply:
We do the exact same for tree path eliding, and the full path is visible in the tooltip. We can add an option if someone requests it, but there is no point in adding additional complexity prematurely. Regardless, a field, not a global variable, would be the solution if ever necessary.